src/scss/phy/featured-profile.scss

```
@@ -3,3 +3,52 @@
 .profile-description {
   padding: 0.5rem 1.5rem 0 1.5rem;
 }
+
+// Detecting a div with any number of featured-profile children
+div:has(> .featured-profile) {
+  @extend .flex-row;
+  @extend .card-deck;
+  @extend .row-cols-1;
+  @extend .row-cols-sm-2;
+  @extend .row-cols-md-3;
+
+  .featured-profile {
+    margin-bottom: 1rem;
+  }
+}
+
+// Detecting a div with exactly two featured-profile children (co-founders)
+div:has(> .featured-profile:first-child:nth-last-child(2)) {
+  @extend .flex-row;
+  @extend .row-cols-1;
+  @extend .row-cols-sm-2;
+
+  .featured-profile {
+    @extend .isaac-tab;
+
+    width: 47.5%;
+    @media (max-width: 575px) {
```

In case we ever change the breakpoint values (defined in `isaac.scss`), we should use the breakpoint functions:
```suggestion
    @include media-breakpoint-down(sm) {
```